src/main/java/com/rescuebites/api/users/controllers/requests/RegisterRequest.java

```diff
@@ -9,24 +10,19 @@
 @PasswordMatches
 public record RegisterRequest (
```

Comment:
Esto es una request para registrar qué cosa? Es mejor que los nombres sean bien declarativos. Por el tipo de request me imagino que esto seria algo mejor


```suggestion
public record UserRegistrationRequest (
```